com.github.wwmm.PulseEffects.json

```diff
@@ -0,0 +1,267 @@
+{
+  "app-id": "com.github.wwmm.PulseEffects",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "3.24",
+  "sdk": "org.gnome.Sdk",
+  "sdk-extensions": ["org.freedesktop.Sdk.Extension.gfortran-62"],
+  "command": "pulseeffects",
+  "rename-desktop-file": "pulseeffects.desktop",
+  "rename-icon": "pulseeffects",
+  "finish-args": [
+    "--socket=wayland",
+    "--socket=pulseaudio",
+    "--own-name=com.github.wwmm.pulseeffects",
```

Comment:
To be honest I am uncertain how this part works. This line was added based on earlier testing and other manifests I found.

So if I replaced that with `"id": "com.github.wwmm.pulseeffects"` or `"app-id": "com.github.wwmm.pulseeffects"` that would work?